tbx/static_src/sass/components/_contact-cta.scss

```diff
@@ -1,58 +1,77 @@
 @use 'config' as *;
 
 .contact-cta {
+    background-color: var(--color--theme-primary);
     display: grid;
-    grid-template-columns: repeat(12, 1fr);
+    grid-template-columns: 1fr;
+    padding: $spacer-small;
+
+    @include media-query(medium) {
+        grid-template-columns: auto 1fr;
+        padding: 50px 70px 50px $spacer-small-plus;
+    }
 
     &__text {
-        grid-column: 1 / span 11;
-        margin-bottom: $spacer-mini-plus;
+        color: var(--color--white);
+        margin-bottom: $spacer-small;
 
-        @include media-query(large) {
+        @include media-query(medium) {
             grid-column: 2 / span 10;
             grid-row: 1;
         }
-    }
-
-    &__avatar {
-        grid-row: 2 / span 3;
 
         @include media-query(large) {
-            margin-right: $spacer-small;
-            grid-row: 1 / span 2;
-            grid-column: 1 / span 1;
+            grid-column: 2 / span 1;
         }
     }
 
-    &__name {
-        color: var(--color--heading);
-        display: block;
-        font-weight: $weight--semibold;
+    &__description {
+        @include font-size(size-four);
+    }
+
+    &__avatar {
+        grid-column: span 1;
+        margin-bottom: $spacer-mini-plus;
+
+        @include media-query(medium) {
+            grid-row: span 2;
+            margin-right: $spacer-small-plus;
+            margin-bottom: 0;
+        }
     }
 
     &__cite {
-        margin-top: ($spacer-mini-plus * 0.5);
-        font-style: normal;
-        grid-column: 3 / span 10;
+        @include font-size(size-five);
+        display: flex;
+        flex-direction: column;
+        margin-bottom: $spacer-mini-plus;
 
-        @include media-query(large) {
-            margin-top: 0;
+        @include media-query(medium) {
+            flex-direction: row;
+            flex-wrap: wrap;
+            grid-column: 2 / span 1;
             grid-row: 2;
-            grid-column: 2 / span 5;
+            margin-bottom: 0;
         }
     }
 
+    &__name {
+        color: var(--color--white);
+        font-weight: $weight--semibold;
+    }
+
+    &__role {
+        color: var(--color--black);
+    }
+
     &__button {
```

Comment:
Suggestion: adding `align-self: flex-start;` to the button will prevent it from growing to the height of the text that's on the left of it:

<img width="714" alt="Screenshot 2025-01-20 at 17 03 44" src="https://github.com/user-attachments/assets/38809ef3-3c25-40b2-ac20-2ce259a202fa" />